src/components/Bubble.tsx

```diff
@@ -417,17 +418,20 @@ const CustomBubble = (
       if (!isHasUrgent) {
         return null;
       }
-
-      return (
-        <View style={styles.content.audibleContainer as StyleProp<ViewStyle>}>
-          <Text style={styles.content.textTick}>Audible Message</Text>
-          <SvgXml
-            style={styles.content.iconMic}
-            xml={getSvg('iconMic')}
-            accessibilityLabel="mic"
-          />
-        </View>
-      );
+      if (urgentMessageComponent) {
+        return urgentMessageComponent;
+      } else {
+        return (
+          <View style={styles.content.audibleContainer as StyleProp<ViewStyle>}>
```

Comment:
The type casting looks needless

```suggestion
          <View style={styles.content.audibleContainer}>
```